test/UT/class/eventManagement/dispatcher.spec.ts

```diff
@@ -0,0 +1,290 @@
+// Import Third-party Dependencies
+import { initRedis, closeRedis, Redis, Channel } from "@myunisoft/redis-utils";
+import * as Logger from "pino";
+import Ajv from "ajv";
+
+// Import Internal Dependencies
+import { Dispatcher } from "../../../../src/index";
+import * as EventsSchemas from "../../schema/index";
+
+// Internal Dependencies Mocks
+const logger = Logger.pino();
+const mockedLoggerError = jest.spyOn(logger, "error");
+const mockedLoggerInfo = jest.spyOn(logger, "info");
+const mockedHandleDispatcherMessages = jest.spyOn(Dispatcher.prototype as any, "handleDispatcherMessages");
+const mockedHandleIncomerMessages = jest.spyOn(Dispatcher.prototype as any, "handleIncomerMessages");
+
+// CONSTANTS
+const ajv = new Ajv();
+
+beforeAll(async() => {
+  await initRedis({
+    port: process.env.REDIS_PORT,
+    host: process.env.REDIS_HOST
+  } as any);
+});
+
+afterAll(async() => {
+  await closeRedis();
+});
+
+describe("Dispatcher without options", () => {
+  let dispatcher: Dispatcher;
+
+  beforeAll(async() => {
+    dispatcher = new Dispatcher();
+
+    Reflect.set(dispatcher, "logger", logger);
+
+    await dispatcher.initialize();
+  });
+
+  afterAll(async() => {
+    await dispatcher.close();
+  });
+
+  beforeEach(() => {
+    mockedLoggerError.mockClear();
+    mockedLoggerInfo.mockClear();
+    mockedHandleDispatcherMessages.mockClear();
+    mockedHandleIncomerMessages.mockClear();
+  });
+
+  test("Dispatcher should be defined", () => {
+    expect(dispatcher).toBeInstanceOf(Dispatcher);
+    expect(dispatcher.prefix).toBe("");
+    expect(dispatcher.privateUuid).toBeDefined();
+  });
+
+  test("Publishing a malformed message, it should log a new Error", async() => {
+    const channel = new Channel({
+      name: "dispatcher"
+    });
+
+    await channel.publish({ foo: "bar" });
+
+    await new Promise((resolve) => setTimeout(resolve, 1_000));
```

Comment:
`import timers from "timers/promises";`
```suggestion
    await timers.setTimeout(1_000)
```